src/picomsg/src/empty_system_events/empty_system_events.cpp

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
libpicomsg shouldn't be involved for anything related with system_events. Move this file to the `kernel_super` directory